pages/CompressPage.py

```diff
@@ -268,7 +276,7 @@ def button_4_leave(e):
             image=button_image_5,
             borderwidth=0,
             highlightthickness=0,
-            command=lambda: startCompress(),
+            command=lambda: self.transcoder.transcode(self.save_path, self.video_path),
```

Comment:
Kindly note that this `transcoder.transcode()` function could accept different container format other than AVI. In the future, a feature could be added to support other formats that supports h263 codec.

Reply:
Noted, adding more datatypes features will be added in the future